ddl/ddl_test.go

```diff
@@ -40,13 +40,13 @@ import (
 )
 
 type DDLForTest interface {
-	// SetHook sets the hook.
+	// SetLogHook sets the hook.
 	SetHook(h Callback)
 	// SetInterceptoror sets the interceptor.
 	SetInterceptoror(h Interceptor)
 }
 
-// SetHook implements DDL.SetHook interface.
+// SetLogHook implements DDL.SetLogHook interface.
```

Comment:
Comment and Function name is not consistent...
I think it can't pass this check https://github.com/pingcap/tidb/pull/10787